.gitignore

```diff
@@ -25,4 +25,7 @@ install_manifest.txt
 #klockwork
 kw_reports/*.tmp
 kw_reports/rdkb/*.tmp
-kw_reports/ugw/*.tmp
\ No newline at end of file
+kw_reports/ugw/*.tmp
+
+#vscode generated files
+.vscode/ipch*
```

Comment:
Typically, files that are generated by developer tools (IDE, editor, ...) should not be in `.gitignore`, but either in the user's global `.gitignore`, or in the per-checkout `.git/info/exclude`.

Reply:
ok